polycommit/src/kzg10/data_structures.rs

```diff
@@ -59,6 +65,10 @@ impl<E: PairingEngine> PCUniversalParams for UniversalParams<E> {
     fn max_degree(&self) -> usize {
         self.powers_of_g.len() - 1
     }
+
+    fn supported_degree_bounds(&self) -> Vec<usize> {
```

Comment:
the call sites that I skimmed looked like iterations, so this should ideally return a `&[usize]` or - better - `impl Iterator<Item = usize>`